tools/wheelmaker.py

```diff
@@ -154,7 +156,7 @@ def arcname_from(name):
         hash = hashlib.sha256()
         size = 0
         with open(real_filename, "rb") as fsrc:
-            with self.open(zinfo, "w") as fdst:
+            with self.open(zinfo, "w", force_zip64=self._force_zip64) as fdst:
```

Comment:
Let's just set force_zip64=True always. I don't think there's a disadvantage to always using zip64? Alternatively, stat() real_filename to get its size, and set it conditionally based on the size.